### ISSUE TITLE
Credentials don't work

### ISSUE THREAD

**Issue by Portland54** [sans-dfir/sift#232]
I downloaded version 3 of the the SIFT vm and managed to get it working but nmthe credentials the site says to use are not letting me access my workstation.

**Comment by ekristen** [comment]
What are you using?

**Comment by Portland54** [comment]
It's running on Ubuntu. It installed and started up just fine.

**Comment by ekristen** [comment]
What credentials are you using?

**Comment by Portland54** [comment]
I typed in the ones I found on the site (sansforensics, forensics) but they
didn't work. I ended up figuring out how to import it but now I'm wondered
if that's a big or it would happen again.

On Feb 22, 2018 11:57 PM, "Erik Kristensen" <notifications@github.com>
wrote:

What credentials are you using?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/sans-dfir/sift/issues/232#issuecomment-367909899>, or mute
the thread
